service/controller/resource/servicemonitor/service/nginx-ingress-controller.go

```diff
@@ -28,22 +28,55 @@ func NginxIngressController(cluster metav1.Object) *promv1.ServiceMonitor {
 				MatchNames: []string{"kube-system"},
 			},
 			Endpoints: []promv1.Endpoint{
-				promv1.Endpoint{
-					Port:   "https",
-					Scheme: "https",
+				{
```

Comment:
Are these changes related to the others in this PR?

Reply:
This is what we have configured in g8s-prometheus and I tried it to check if the networking issue was related. Not sure if I should remove it

Reply:
I removed those values for now